app/views.py

```diff
@@ -29,15 +42,20 @@ def index_page():
         for idx in results:
             print(TextToCheck_List[idx])
             word = TextToCheck_List[idx]
-            recommendations.append((word, word_candidates(word)))
-            print(recommendations)
+            if langSelect == 'English':
+                recommendations.append((word, word_candidates(word)))
+            else:
+                print("Add other lang recommendations")
+                recommendations.append((word, ''))
+            # print(recommendations)
```

Comment:
Is there a reason this is commented out? Is it no longer needed? If not, can it be deleted?

Reply:
removed it